docs/crocks/Pair.md

```diff
@@ -0,0 +1,77 @@
+# Pair
+
+`Pair c a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Pair` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`type`](#type) | [`ap`](#ap), [`bimap`](#bimap), [`chain`](#chain), [`concat`](#concat), [`equals`](#equals), [`fst`](#fst), [`inspect`](#inspect),  [`map`](#map), [`merge`](#merge), [`of`](#of), [`type`](#type), [`snd`](#snd), [`swap`](#swap), [`value`](#value) |
+
+## Constructor
+
+### type
+
+`() -> String`
+
+## Instance
+
+### ap
+
+`Pair m, Semigroup s => m s (a -> b) ~> m s a -> m s b`
+
+### bimap
+
+`Pair m => m c a ~> (c -> d) -> (a -> b) -> m d b`
+
+### chain
+
+`Pair m, Semigroup s => m s a ~> (a -> m s b) -> m s b`
+
+### concat
+
+`Pair m, Semigroup sc, Semigroup sa => m sc sa ~> m sc sa -> m sc sa`
+
+### equals
+
+`a -> Boolean`
+
+### fst
+
+`Pair m => m c a ~> () -> c`
+
+### inspect
+
+`() -> String`
+
+### map
+
+`Pair m => m c a ~> (a -> b) -> m c b`
+
+### merge
+
+`Pair m => m c a ~> (a -> b -> c) -> a b -> c`
+
+### type
+
+`() -> String`
+
+### snd
+
+`Pair m => m c a ~> () -> a`
+
+### swap
+
+`Pair m => m c a ~> m c a -> m a c`
```

Comment:
The inputs to this are not right.
(check the source code)
pretty sure it takes functions.